judge/views/contests.py

```diff
@@ -485,6 +490,71 @@ def render(self):
         return response
 
 
+class ContestStats(TitleMixin, ContestMixin, DetailView):
+    template_name = 'contest/stats.html'
+
+    def get_title(self):
+        return _('%s Statistics') % self.object.name
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        if not self.object.ended:
```

Comment:
I'm not sure about how sensitive this data is. It's already fully public for non-hidden-scoreboard contests, so maybe it should be visible before the end of the contest? (At least, it should be visible if `self.object.is_editable_by(...)`)

Everything else looks fine, btw.

Reply:
The contest statistics page exposes problem names, which are hidden from users who are not in the contest. A commit has been added to allow contest organizers to see the statistics before the end of the contest.